src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
+  } else if (days >= 3) {
+    finalPrice = 40 * days - 20;
```

Comment:
Similarly, you can return the calculated price directly from within the `else if` block.